applications/utils.py

```diff
@@ -73,6 +76,14 @@ def export_berth_applications_as_xlsx(applications):
                     value = application.boat_type.name
                 elif attr_name == "berth_switch" and application.berth_switch:
                     value = parse_berth_switch_str(application.berth_switch)
+                elif attr_name == "berth_switch_reason":
```

Comment:
```python
elif attr_name == "berth_switch_reason" and application.berth_switch:
    value = application.berth_switch.reason.title if application.berth_switch.reason else "---"
```
?

Reply:
Yeah, I thought about this, but if `application.berth_switch` is `None`, it will try to get the `berth_switch_reason` from the `Application` instance and throw an error because it doesn't have that attribute.

```python
else:
    value = getattr(application, attr_name)
```

Reply:
ah, right! okay then 👍